### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by gricey432**
Fixed a couple of small PEP8 violations and removed the `requirements.txt` because it didn't seem to do anything and was messing with pycharm.

Split out of #266

### CODE REVIEW

File: uqcsbot/scripts/welcome.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
-# Welcomes new users to UQCS Slack and check for member milestones
+"""
+Welcomes new users to UQCS Slack and check for member milestones
+"""
 from uqcsbot import bot
 import asyncio
 
-MEMBER_MILESTONE = 50;  # Number of members between posting a celebration
-MESSAGE_PAUSE = 2.5;   # Number of seconds between sending bot messages
```
Comment (mitchmcdee) [inline uqcsbot/scripts/welcome.py]: lol

File: requirements.txt [hunk 2]
```diff
@@ -1 +0,0 @@
--e .
```
Comment (mitchmcdee) [inline requirements.txt]: Does this affect installation at all? and if that's the only line, should we just delete the file?

Comment (CameronAavik) [reply requirements.txt]: I remember tom telling me this was necessary to keep, but I can't remember why

Comment (gricey432) [reply requirements.txt]: I removed it and it installed just fine for me, I've never seen anything like this before. @TRManderson ?

Comment (TRManderson) [reply requirements.txt]: The answer is "it depends".

@gricey432, you're very used to doing `pip install -e .` from work stuff, but that's not the normal workflow for most python devs.

people using `pipenv` need the `requirements.txt`
people using `requirements.txt` need it
people using `setup.py` are covered
people using `pip install .` are covered

Comment (gricey432) [reply requirements.txt]: But those first two methods won't work with tests_requires right? So having it there is a bit confusing.

Comment (gricey432) [reply requirements.txt]: The `README.md` has been changed to tell people to use `pip`, so I'd say we don't need this anymore. Keeping it as simple as possible for a first-year to look around the repo and figure out what stuff is for

**Pull request merged**